mycart/mycartstyles.css

```diff
@@ -1,145 +1,43 @@
-    gap: 30px;
-}
-.nav-right-container i {
-    padding-top: 10px;
-}
-.badge1 {
-    position: absolute;
-    font-size: 12px;
-    top: 14px;
-    right: 190px;
-    padding: 0px 6px;
-    border-radius: 50%;
-    background-color: rgb(250, 53, 53);
-    color: white;
+    padding: 0rem 1rem 1rem 1rem;
 }
 .badge2 {
     position: absolute;
-    font-size: 12px;
-    top: 14px;
-    right: 135px;
-    padding: 0px 6px;
+    font-size: 1rem;
+    top: 1.15rem;
+    right: 8.5rem;
+    padding: 0rem 0.34rem;
     border-radius: 50%;
-    background-color: rgb(250, 53, 53);
-    color: white;
+    background-color: var(--heart-icon-color);
+    color: var(--background-color);
 }
-
-
 /* front end */
-.card {
-    margin: 50px 150px;
-    width: 630px;
-    height: 350px;
-    background-color: white;
-    display: flex;
-    flex-wrap: wrap;
-    box-shadow: -9px -3px 73px -43px rgba(0, 0, 0, 0.75);
-    -webkit-box-shadow: -9px -3px 73px -43px rgba(0, 0, 0, 0.75);
-    -moz-box-shadow: -9px -3px 73px -43px rgba(0, 0, 0, 0.75);
-}
-.card img {
-    position: relative;
-    width: 300px;
-    height: 349px;
-    padding-right: 20px;
-}
-.card .img-label {
-    margin-top: 30px;
-    font-size: 22px;
-    color: black;
-    margin-right: 200px;
-}
-.card .price {
-    font-size: 24px;
-    font-weight: bolder;
-}
-.card .strike {
-    font-size: 20px;
-    color: #888;
-}
-.card .off {
-    font-size: 18px;
-    font-weight: bold;
-    color: rgb(87, 87, 87);
-}
-.add .sub {
-    border-radius: 50%;
-    padding: 2px 6px;
-    border: 1px solid black;
-}
-.quantity .add {
-    border-radius: 50%;
-    border: 1px solid black;
-    padding: 1px 6px;
-    margin: 3px;
-}
-.quantity .total {
-    border-radius: 2px;
-    border: 1px solid black;
-    padding: 3px 20px;
-    margin: 3px;
-}
-.quantity {
-    margin-bottom: 20px;
-}
-.remove-btn,
-.move-btn {
-    width: 313px;
-    height: 40px;
-    margin-top: 10px;
-}
-.remove-link{
-    color: white;
+.remove-link {
+    color: var(--background-color);
 }
 
 /*  */
-.price-container {
-    display: flex;
-    flex-direction: column;
-    margin: 50px 10px;
-    width: 400px;
-    height: 450px;
-    background-color: white;
-    box-shadow: -9px -3px 73px -50px rgba(0, 0, 0, 0.75);
-    -webkit-box-shadow: -9px -3px 73px -50px rgba(0, 0, 0, 0.75);
-    -moz-box-shadow: -9px -3px 73px -50px rgba(0, 0, 0, 0.75);
-    padding: 30px;
-}
-.price-container label {
-    display: flex;
-    justify-content: space-between;
-    padding-top: 10px;
-}
-.price-title {
-    font-size: 24px;
-    font-weight: 700;
-    border-bottom: 1px solid black;
-}
-.total {
-    font-weight: bold;
-    border-top: 1px solid black;
-    border-bottom: 1px solid black;
-}
-.price-footer {
-    padding: 5px;
-}
-
-.grid-container{
+.grid-container {
     display: grid;
     grid-template-columns: auto auto;
-}
\ No newline at end of file
+}
+.move-to-cart{
+    width: 20rem;
+    padding: 0.4rem;
+    margin-top: 0.90rem;
```

Comment:
You can use px for these values

Reply:
yeah we can add both Thanks For Review @ranitasahaa1230